internal/pkg/policy/policy_output.go

```
@@ -74,13 +74,17 @@ func (p *Output) prepareElasticsearch(
 	zlog zerolog.Logger,
 	bulker bulk.Bulk,
 	agent *model.Agent,
-	outputMap smap.Map) error {
+	outputMap map[string]map[string]interface{}) error {
 	// The role is required to do api key management
 	if p.Role == nil {
 		zlog.Error().
 			Msg("policy does not contain required output permission section")
 		return ErrNoOutputPerms
 	}
+	if _, ok := outputMap[p.Name]; !ok {
```

fail fast here instead of a possible panic on line 245